<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComfortQuestionnaire</class>
 <widget class="QWidget" name="ComfortQuestionnaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comfort Questionnaire</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="answersGroup">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>310</width>
     <height>185</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Would you please fill out the questionnaire?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QGridLayout" name="answers">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="answer2" native="true">
      <widget class="QRadioButton" name="radioButtonAnswer2">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer2">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No/Later</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="answer3" native="true">
      <widget class="QRadioButton" name="radioButtonAnswer3">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer3">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Deactivate
questionnaire</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="answer1" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QRadioButton" name="radioButtonAnswer1">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer1">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="answer5" native="true">
      <widget class="QRadioButton" name="radioButtonAnswer5">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer5">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Older than
61</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="answer4" native="true">
      <widget class="QRadioButton" name="radioButtonAnswer4">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer4">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>51-60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="answer6" native="true">
      <widget class="QRadioButton" name="radioButtonAnswer6">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>50</y>
         <width>26</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelAnswer6">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>5</y>
         <width>94</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Very
comforting</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
